graph/api/src/main/scala/org/apache/spark/graph/api/CypherSession.scala

```diff
@@ -0,0 +1,139 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
```

Comment:
Could you remove this new line? The other license header seems to have the same issue.

Reply:
```
$ dev/scalastyle
[error] /home/jenkins/workspace/SparkPullRequestBuilder/graph/api/src/main/scala/org/apache/spark/graph/api/CypherSession.scala:16: Header does not match expected text
[error] /home/jenkins/workspace/SparkPullRequestBuilder/graph/api/src/main/scala/org/apache/spark/graph/api/PropertyGraph.scala:16: Header does not match expected text
[error] /home/jenkins/workspace/SparkPullRequestBuilder/graph/api/src/main/scala/org/apache/spark/graph/api/GraphElementFrame.scala:16: Header does not match expected text
[error] /home/jenkins/workspace/SparkPullRequestBuilder/graph/api/src/main/scala/org/apache/spark/graph/api/PropertyGraphType.scala:16: Header does not match expected text
```